src/test/java/marquez/db/DatasetDaoTest.java

```diff
@@ -61,13 +66,31 @@ public static void setUpOnce() {
     datasetDao = jdbi.onDemand(DatasetDao.class);
   }
 
+  @Before
+  public void setup() {
+    datasourceRow = newDatasourceRow();
+    namespaceRow = newNamespaceRow();
+
+    namespaceDao.insert(namespaceRow);
+    datasourceDao.insert(datasourceRow);
+  }
+
+  @Test
+  public void testGetEmptySetOfRowsByUUID() {
+    Optional<DatasetRow> returnedRow = datasetDao.findBy(UUID.randomUUID());
+    assertThat(returnedRow).isEmpty();
+  }
+
+  @Test
+  public void testFindAllWithEmptySetOfRows() {
```

Comment:
_Minor_: `testFindAll_noRowsFound`. And a more natural flow, for readability, would be to move this test below `testFindAll`

Reply:
Done